plugin/trace/README.md

```diff
@@ -57,7 +57,7 @@ trace tracinghost:9253
 
 or
 
-~~~ corefile
```

Comment:
why this change?

Reply:
```
readme_test.go:81: Failed to start server with ../plugin/trace/README.md, for input "lookup : no such host":
        . {
            trace zipkin tracinghost:9253
        }
```

Reply:
https://github.com/openzipkin/zipkin-go/blob/62dc8b26c05e0e8b88eb7536eff92498e65bbfc3/endpoint.go#L55

Reply:
can we make it localhost?

Reply:
done